src/Components/DropdownItem.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import cn from 'classnames';
+import { Person } from '../types/Person';
+
+const PERSON_SEX_MAN = 'm';
+const PERSON_SEX_WOMAN = 'f';
+
+type Props = {
+  person: Person
+  onSelectPerson: (person: Person) => void;
+};
+
+export const DropdownItem: React.FC<Props> = ({ person, onSelectPerson }) => (
+  // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+  <div
+    className="dropdown-item"
+    style={{ cursor: 'pointer' }}
```

Comment:
Try avoiding mixing inline styles and styling with classes, as this would be hard to support